variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/VariantSource.java

```diff
@@ -25,24 +26,76 @@
  */
 public class VariantSource implements IVariantSource {
 
+    public final static String FILEID_FIELD = "fid";
```

Comment:
these fields are already in the uk.ac.ebi.eva.commons.mongodb.entities.VariantSourceMongo class. The changes here would require to add a mongo dependency to the variation-commons core submodule. this class should remain intact